js/targetCowFunctions.js

```diff
@@ -0,0 +1,85 @@
+$('body').on('click', '.js_clickable_cow', function(e) {
+    save_cow(e)
+})
+
+/**
+ * when the cow is clicked, remove the class js_clickable_cow, add class rocket_cow and execute rocket cow function
+ */
+function save_cow(e) {
+    var clicked_cow = e.target
```

Comment:
might as well make this a jquery variable rather than wrapping it in brackets twice below